webpack.config.js

```diff
@@ -205,6 +205,30 @@ module.exports = function(env) {
         )
     );
 
+    if (env.hash)
```

Comment:
env.hash is set by some scripts in package.json (e.g., when you run npm run build-hash)

Reply:
Why do you need this here?

Reply:
blocktrades uses oauth for its logins. oauth requires you to specify a redirect uri where you get returned after successfully logging in/out. blocktrades uses the deposit/withdrawal page as that redirect uri.  when using the browser router, this url is `/deposit-withdraw`; when using the hash router, it is `/#/deposit-withdraw`.  according to the oauth spec, this uri cannot contain any hash fragments, so we cannot register the redirect uri understood by the hash router with our oauth server.  instead, we register `/deposit-withdraw` for our redirect uri which will work when using the browser router.  when using the hash router (github pages), it would normally fail because the server does not automatically rewrite this request to load index.html. instead, github pages tries to load the file `/deposit-withdraw` which doesn't exist, then returns a 404 not found, causing blocktrades login to fail.  to work around this, we create a file the github pages server will find when it looks for `/deposit-withdraw` that will redirect to the hash router's equivalent `/#/deposit-withdraw`

Reply:
Thanks, please boil that into a comment inside the code, reference to the redirect file etc.

Reply:
@sschiessl-bcp Comment is added.